internal/state/store_test.go

```
@@ -13,52 +13,81 @@ func TestStoreUpdateResultSuccessAndFailure(t *testing.T) {
 	}
 }
 
 func TestStoreHistorySize(t *testing.T) {
 	store := NewStore([]config.TargetConfig{{Name: "example"}}, 100*time.Millisecond)
 	store.historySize = 2
 
-	store.UpdateResult("example", ping.Result{Success: true, RTT: 10 * time.Millisecond})
-	store.UpdateResult("example", ping.Result{Success: true, RTT: 11 * time.Millisecond})
-	store.UpdateResult("example", ping.Result{Success: true, RTT: 12 * time.Millisecond})
+	for _, rtt := range []time.Duration{10, 11, 12} {
+		store.UpdateResult("example", ping.Result{Success: true, RTT: rtt * time.Millisecond})
```

golangci-lint (2.12.2)

[error] Multiplication of durations: `rtt * time.Millisecond`

(durationcheck)